.github/workflows/macos-linux-pip.yml

```
@@ -40,7 +40,7 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu", "macos"]
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
```

```suggestion
        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```

python 3.8 does not work on github macos arm runners

---

Oh, wait, it is already excluded